src/resources/elements/timeLeft/timeLeft.scss

```diff
@@ -32,12 +35,12 @@
       }
 
       &.farOff {
-        color: $White;
+        color: #cecece;
```

Comment:
Suggestion to always use scss variable for any hex-color.
Could also be just defined on top of this file.

Here, it could be just `$FarOffColor`